DiagnosticsExtension/Models/ConnectionStringValidator/Exceptions/ConnectionStringResponseUtility.cs

```diff
@@ -0,0 +1,128 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="ConnectionStringResponseUtility.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using Microsoft.WindowsAzure.Storage;
+
+namespace DiagnosticsExtension.Models.ConnectionStringValidator.Exceptions
+{
+    public static class ConnectionStringResponseUtility
+    {
+        #region string constants for network validator
+        public const string ClientId = "__clientId";
+        public const string Credential = "__credential";
+        public const string ServiceUriMissed = "ServiceUriMissed";
+        public const string ValidCredentialValue = "managedidentity";
+        public const string FullyQualifiedNamespace = "__fullyQualifiedNamespace";
+        public const string ManagedIdentityCredentialMissing = "ManagedIdentityCredentialMissing";
+        #endregion
+
+        public static ConnectionStringValidationResult EvaluateResponseStatus(Exception e, ConnectionStringType type, ConnectionStringValidationResult.ManagedIdentityType identityType)
```

Comment:
I don't see where "ManagedIdentityException" type is checked in this function?  What is the point of having this new exception type?

Reply:
fixed